test_bgp_default_originate_timer.py

```diff
@@ -0,0 +1,166 @@
+    topo.router("r2").iface_to("s1").ip4.append("192.168.1.2/24")
+    topo.router("r1").iface_to("s2").ip4.append("192.168.2.1/24")
+    topo.router("r3").iface_to("s1").ip4.append("192.168.2.2/24")
+
+
+class Configs(FRRConfigs):
+    routers = ["r1", "r2", "r3"]
+
+    zebra = """
+    #% extends "boilerplate.conf"
+    #% block main
+    #%   for iface in router.ifaces
+    interface {{ iface.ifname }}
+     ip address {{ iface.ip4[0] }}
+    !
+    #%   endfor
+    #% endblock
+    """
+
+    bgpd = """
+    #% extends "boilerplate.conf"
+    #% block main
+    #%   if router.name == 'r1'
+    router bgp 65001
+     no bgp ebgp-requires-policy
+     bgp default-originate timer 3600
+     neighbor {{ routers.r2.iface_to('s1').ip4[0].ip }} remote-as external
+     neighbor {{ routers.r2.iface_to('s1').ip4[0].ip }} timers 1 3
+     neighbor {{ routers.r2.iface_to('s1').ip4[0].ip }} timers connect 1
+     neighbor {{ routers.r3.iface_to('s1').ip4[0].ip }} remote-as external
+     neighbor {{ routers.r3.iface_to('s1').ip4[0].ip }} timers 1 3
+     neighbor {{ routers.r3.iface_to('s1').ip4[0].ip }} timers connect 1
+     address-family ipv4
+      neighbor {{ routers.r2.iface_to('s1').ip4[0].ip }} default-originate route-map default
+     exit-address-family
+    !
+    bgp community-list standard r3 seq 5 permit 65003:1
+    !
+    route-map default permit 10
+     match community r3
+    exit
+    #%   elif router.name == 'r2'
+    router bgp 65002
+     no bgp ebgp-requires-policy
+     neighbor {{ routers.r1.iface_to('s1').ip4[0].ip }} remote-as external
+     neighbor {{ routers.r1.iface_to('s1').ip4[0].ip }} timers 1 3
+     neighbor {{ routers.r1.iface_to('s1').ip4[0].ip }} timers connect 1
+    !
+    #%   elif router.name == 'r3'
+    router bgp 65003
+     no bgp ebgp-requires-policy
+     neighbor {{ routers.r1.iface_to('s2').ip4[0].ip }} remote-as external
+     neighbor {{ routers.r1.iface_to('s2').ip4[0].ip }} timers 1 3
+     neighbor {{ routers.r1.iface_to('s2').ip4[0].ip }} timers connect 1
+     address-family ipv4 unicast
+      redistribute connected route-map r1
+     exit-address-family
+    !
+    route-map r1 permit 10
+     set community 65003:1
+    exit
+    #%   endif
+    #% endblock
+    """
+
+
+class BGPDefaultOriginateTimer(TestBase, AutoFixture, topo=topology, configs=Configs):
+    @topotatofunc
+    def bgp_default_received_from_r1(self, _, r1, r2):
+        expected = {
+            "paths": [
+                {
+                    "nexthops": [
+                        {
+                            "hostname": "r1",
+                            "ip": str(r1.iface_to("s1").ip4[0].ip),
+                        }
+                    ],
+                }
+            ],
+        }
+
+        yield from AssertVtysh.make(
+            r2,
+            "bgpd",
+            f"show bgp ipv4 unicast 0.0.0.0/0 json",
+            maxwait=10.0,
+            compare=expected,
```

Comment:
in the original topotest this is a "negative" check, it checks the route does NOT exist — need to do something similar here.

Good approach would be:
1. wait for some _other_ route to check the BGP session is up
2. then check default route is empty/absent